frontend/src/mocks/msw/handlers/user.ts

```diff
@@ -34,18 +32,6 @@ export const getUser = ({
   return rest.get<never, never, UserDto | ErrorDto>(
     '/api/v3/user',
     (_req, res, ctx) => {
-      // Check if the user is authenticated in this session
-      const isAuthenticated = localStorage.getItem(LOGGED_IN_KEY)
-      if (!isAuthenticated) {
-        // If not authenticated, respond with a 403 error
-        return res(
-          ctx.delay(),
-          ctx.status(403),
-          ctx.json({
-            message: 'Not authorized',
-          }),
-        )
-      }
```

Comment:
If the case where the user is not authenticated is not handled in this mock handler function, is there a necessity for the return type to be `UserDto | ErrorDto` instead of just `UserDto`? This is because the function will always return a `UserDto` object.